.github/workflows/test-macos.yml

```diff
@@ -40,9 +40,7 @@ jobs:
     if: github.event.pull_request.draft == false
     strategy:
       fail-fast: false
-      matrix:
-        macos-version: [macos-13, macos-14]
-    runs-on: ${{ matrix.macos-version }}
+    runs-on: macos-14
```

Comment:
nit: I guess the commit message should be `drop deprecated x64 macOS runner` or something like that, as we're not really updating anything, just removing one